doc/network_file_desc.rst

```diff
@@ -0,0 +1,196 @@
+:orphan:
+
+.. _network_file_content:
+
+====================
+Network File Content
+====================
+
+File Content Description
+------------------------
+
+    object_type : str
```

Comment:
you have an extra tab level which is making the rendering appear as if it's quoted:

https://output.circle-artifacts.com/output/job/2e226fdd-4b37-4b45-84b2-23faa4488796/artifacts/0/html/network_file_desc.html